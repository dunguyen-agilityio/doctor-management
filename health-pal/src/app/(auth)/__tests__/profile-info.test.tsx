import { render } from '@utils-test'

import ProfileInfo from '../profile-info'

describe('<ProfileInfo />', () => {
  it('matches snapshot after loading', async () => {
    const { toJSON } = render(<ProfileInfo />)
    expect(toJSON()).toMatchSnapshot()
  })
})
