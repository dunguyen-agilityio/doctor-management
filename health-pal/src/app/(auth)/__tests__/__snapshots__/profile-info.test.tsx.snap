// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileInfo /> matches snapshot after loading 1`] = `
[
  <View
    aria-label="Notifications (F8)"
    pointerEvents="box-none"
    role="region"
    style={
      {
        "bottom": 0,
        "flexDirection": "column",
        "left": 0,
        "maxWidth": "100%",
        "position": "absolute",
        "right": 0,
        "top": 0,
        "zIndex": 100000,
      }
    }
    tabIndex={-1}
  >
    <View
      focusable={false}
      pointerEvents="box-none"
      style={
        {
          "flexDirection": "column",
          "maxWidth": "100%",
          "position": "absolute",
        }
      }
    />
  </View>,
  <View
    style={
      {
        "flexDirection": "column",
        "height": 812,
      }
    }
  >
    <RCTScrollView
      contentContainerStyle={
        {
          "flexGrow": 1,
        }
      }
      keyboardShouldPersistTaps="handled"
      scrollEnabled={true}
    >
      <View>
        <View
          style={
            {
              "backgroundColor": "white",
              "flex": 1,
              "flexDirection": "column",
              "gap": 24,
              "paddingLeft": 24,
              "paddingRight": 24,
            }
          }
        >
          <View
            style={
              {
                "backgroundColor": "white",
                "flexDirection": "column",
                "gap": 20,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "white",
                  "flexDirection": "row",
                  "justifyContent": "center",
                }
              }
            >
              <View
                style={[]}
              />
            </View>
            <View
              style={[]}
            >
              <View
                style={[]}
              >
                <TextInput
                  accessibilityHint="Enter your full name for your profile"
                  accessibilityRole="text"
                  aria-label="Full name"
                  editable={true}
                  focusVisibleStyle={{}}
                  focusable={true}
                  name="name"
                  onBlur={[Function]}
                  onChangeText={[Function]}
                  onFocus={[Function]}
                  onSubmitEditing={[Function]}
                  placeholder="Name"
                  placeholderTextColor="#9ca3af"
                  returnKeyType="next"
                  style={
                    {
                      "backgroundColor": "#f9fafb",
                      "borderBottomColor": "#d1d5db",
                      "borderBottomLeftRadius": 8,
                      "borderBottomRightRadius": 8,
                      "borderBottomWidth": 1,
                      "borderLeftColor": "#d1d5db",
                      "borderLeftWidth": 1,
                      "borderRightColor": "#d1d5db",
                      "borderRightWidth": 1,
                      "borderStyle": "solid",
                      "borderTopColor": "#d1d5db",
                      "borderTopLeftRadius": 8,
                      "borderTopRightRadius": 8,
                      "borderTopWidth": 1,
                      "color": "#374151",
                      "flex": 1,
                      "fontFamily": "Inter_400Regular",
                      "fontSize": 14,
                      "height": 45,
                      "minWidth": 0,
                      "paddingBottom": 12,
                      "paddingLeft": 16,
                      "paddingRight": 16,
                      "paddingTop": 12,
                    }
                  }
                  textContentType="name"
                />
              </View>
            </View>
            <View
              style={[]}
            >
              <View
                style={[]}
              >
                <TextInput
                  accessibilityHint="Enter your nickname for your profile"
                  accessibilityRole="text"
                  aria-label="Nickname"
                  focusVisibleStyle={{}}
                  focusable={true}
                  name="nickname"
                  onBlur={[Function]}
                  onChangeText={[Function]}
                  onFocus={[Function]}
                  onSubmitEditing={[Function]}
                  placeholder="Nickname"
                  placeholderTextColor="#9ca3af"
                  returnKeyType="next"
                  style={
                    {
                      "backgroundColor": "#f9fafb",
                      "borderBottomColor": "#d1d5db",
                      "borderBottomLeftRadius": 8,
                      "borderBottomRightRadius": 8,
                      "borderBottomWidth": 1,
                      "borderLeftColor": "#d1d5db",
                      "borderLeftWidth": 1,
                      "borderRightColor": "#d1d5db",
                      "borderRightWidth": 1,
                      "borderStyle": "solid",
                      "borderTopColor": "#d1d5db",
                      "borderTopLeftRadius": 8,
                      "borderTopRightRadius": 8,
                      "borderTopWidth": 1,
                      "color": "#374151",
                      "flex": 1,
                      "fontFamily": "Inter_400Regular",
                      "fontSize": 14,
                      "height": 45,
                      "minWidth": 0,
                      "paddingBottom": 12,
                      "paddingLeft": 16,
                      "paddingRight": 16,
                      "paddingTop": 12,
                    }
                  }
                />
              </View>
            </View>
            <View
              style={[]}
            >
              <View
                style={[]}
              >
                <TextInput
                  accessibilityHint="Enter your email address for your profile"
                  accessibilityRole="text"
                  aria-label="Email address"
                  editable={true}
                  focusVisibleStyle={{}}
                  focusable={true}
                  name="email"
                  onBlur={[Function]}
                  onChangeText={[Function]}
                  onFocus={[Function]}
                  onSubmitEditing={[Function]}
                  placeholder="Email"
                  placeholderTextColor="#9ca3af"
                  returnKeyType="next"
                  style={
                    {
                      "backgroundColor": "#f9fafb",
                      "borderBottomColor": "#d1d5db",
                      "borderBottomLeftRadius": 8,
                      "borderBottomRightRadius": 8,
                      "borderBottomWidth": 1,
                      "borderLeftColor": "#d1d5db",
                      "borderLeftWidth": 1,
                      "borderRightColor": "#d1d5db",
                      "borderRightWidth": 1,
                      "borderStyle": "solid",
                      "borderTopColor": "#d1d5db",
                      "borderTopLeftRadius": 8,
                      "borderTopRightRadius": 8,
                      "borderTopWidth": 1,
                      "color": "#374151",
                      "flex": 1,
                      "fontFamily": "Inter_400Regular",
                      "fontSize": 14,
                      "height": 45,
                      "minWidth": 0,
                      "paddingBottom": 12,
                      "paddingLeft": 16,
                      "paddingRight": 16,
                      "paddingTop": 12,
                    }
                  }
                />
              </View>
            </View>
            <View
              accessibilityHint="Select a date from the calendar"
              aria-expanded={false}
              aria-label="Date picker"
              focusVisibleStyle={{}}
              focusable={true}
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="dialog"
              style={
                {
                  "alignItems": "center",
                  "backgroundColor": "hsla(0, 0%, 100%, 1)",
                  "borderBottomColor": "transparent",
                  "borderBottomWidth": 1,
                  "borderLeftColor": "transparent",
                  "borderLeftWidth": 1,
                  "borderRightColor": "transparent",
                  "borderRightWidth": 1,
                  "borderStyle": "solid",
                  "borderTopColor": "transparent",
                  "borderTopWidth": 1,
                  "flexDirection": "row",
                  "flexWrap": "nowrap",
                  "height": 48,
                  "justifyContent": "center",
                  "paddingBottom": 0,
                  "paddingLeft": 0,
                  "paddingRight": 0,
                  "paddingTop": 0,
                }
              }
            >
              <View
                style={[]}
              >
                <View
                  style={[]}
                >
                  <View
                    style={[]}
                  >
                    <RNSVGSvgView
                      align="xMidYMid"
                      bbHeight="19"
                      bbWidth="18"
                      fill="none"
                      focusable={false}
                      height="19"
                      meetOrSlice={0}
                      minX={0}
                      minY={0}
                      style={
                        [
                          {
                            "backgroundColor": "transparent",
                            "borderWidth": 0,
                          },
                          {
                            "flex": 0,
                            "height": 19,
                            "width": 18,
                          },
                        ]
                      }
                      testID="left-icon"
                      vbHeight={19}
                      vbWidth={18}
                      width="18"
                    >
                      <RNSVGGroup
                        fill={null}
                        propList={
                          [
                            "fill",
                          ]
                        }
                      >
                        <RNSVGPath
                          d="M6 2V4.25"
                          fill={
                            {
                              "payload": 4278190080,
                              "type": 0,
                            }
                          }
                          propList={
                            [
                              "stroke",
                              "strokeWidth",
                              "strokeLinecap",
                              "strokeLinejoin",
                              "strokeMiterlimit",
                            ]
                          }
                          stroke={
                            {
                              "payload": 4288455599,
                              "type": 0,
                            }
                          }
                          strokeLinecap={1}
                          strokeLinejoin={1}
                          strokeMiterlimit={10}
                          strokeWidth="1.5"
                        />
                        <RNSVGPath
                          d="M12 2V4.25"
                          fill={
                            {
                              "payload": 4278190080,
                              "type": 0,
                            }
                          }
                          propList={
                            [
                              "stroke",
                              "strokeWidth",
                              "strokeLinecap",
                              "strokeLinejoin",
                              "strokeMiterlimit",
                            ]
                          }
                          stroke={
                            {
                              "payload": 4288455599,
                              "type": 0,
                            }
                          }
                          strokeLinecap={1}
                          strokeLinejoin={1}
                          strokeMiterlimit={10}
                          strokeWidth="1.5"
                        />
                        <RNSVGPath
                          d="M2.625 7.3175H15.375"
                          fill={
                            {
                              "payload": 4278190080,
                              "type": 0,
                            }
                          }
                          propList={
                            [
                              "stroke",
                              "strokeWidth",
                              "strokeLinecap",
                              "strokeLinejoin",
                              "strokeMiterlimit",
                            ]
                          }
                          stroke={
                            {
                              "payload": 4288455599,
                              "type": 0,
                            }
                          }
                          strokeLinecap={1}
                          strokeLinejoin={1}
                          strokeMiterlimit={10}
                          strokeWidth="1.5"
                        />
                        <RNSVGPath
                          d="M15.75 6.875V13.25C15.75 15.5 14.625 17 12 17H6C3.375 17 2.25 15.5 2.25 13.25V6.875C2.25 4.625 3.375 3.125 6 3.125H12C14.625 3.125 15.75 4.625 15.75 6.875Z"
                          fill={
                            {
                              "payload": 4278190080,
                              "type": 0,
                            }
                          }
                          propList={
                            [
                              "stroke",
                              "strokeWidth",
                              "strokeLinecap",
                              "strokeLinejoin",
                              "strokeMiterlimit",
                            ]
                          }
                          stroke={
                            {
                              "payload": 4288455599,
                              "type": 0,
                            }
                          }
                          strokeLinecap={1}
                          strokeLinejoin={1}
                          strokeMiterlimit={10}
                          strokeWidth="1.5"
                        />
                        <RNSVGPath
                          d="M8.99686 10.775H9.00359"
                          fill={
                            {
                              "payload": 4278190080,
                              "type": 0,
                            }
                          }
                          propList={
                            [
                              "stroke",
                              "strokeWidth",
                              "strokeLinecap",
                              "strokeLinejoin",
                            ]
                          }
                          stroke={
                            {
                              "payload": 4288455599,
                              "type": 0,
                            }
                          }
                          strokeLinecap={1}
                          strokeLinejoin={1}
                          strokeWidth="2"
                        />
                        <RNSVGPath
                          d="M6.22049 10.775H6.22723"
                          fill={
                            {
                              "payload": 4278190080,
                              "type": 0,
                            }
                          }
                          propList={
                            [
                              "stroke",
                              "strokeWidth",
                              "strokeLinecap",
                              "strokeLinejoin",
                            ]
                          }
                          stroke={
                            {
                              "payload": 4288455599,
                              "type": 0,
                            }
                          }
                          strokeLinecap={1}
                          strokeLinejoin={1}
                          strokeWidth="2"
                        />
                        <RNSVGPath
                          d="M6.22049 13.025H6.22723"
                          fill={
                            {
                              "payload": 4278190080,
                              "type": 0,
                            }
                          }
                          propList={
                            [
                              "stroke",
                              "strokeWidth",
                              "strokeLinecap",
                              "strokeLinejoin",
                            ]
                          }
                          stroke={
                            {
                              "payload": 4288455599,
                              "type": 0,
                            }
                          }
                          strokeLinecap={1}
                          strokeLinejoin={1}
                          strokeWidth="2"
                        />
                      </RNSVGGroup>
                    </RNSVGSvgView>
                  </View>
                  <TextInput
                    accessibilityHint="Select your date of birth for your profile"
                    accessibilityRole="text"
                    aria-label="Date of birth"
                    defaultValue=""
                    editable={false}
                    focusVisibleStyle={{}}
                    focusable={true}
                    name="dateOfBirth"
                    onBlur={[Function]}
                    onChangeText={[Function]}
                    onFocus={[Function]}
                    placeholder="Date of birth"
                    placeholderTextColor="#9ca3af"
                    pointerEvents="none"
                    style={
                      {
                        "backgroundColor": "#f9fafb",
                        "borderBottomColor": "#d1d5db",
                        "borderBottomLeftRadius": 8,
                        "borderBottomRightRadius": 8,
                        "borderBottomWidth": 1,
                        "borderLeftColor": "#d1d5db",
                        "borderLeftWidth": 1,
                        "borderRightColor": "#d1d5db",
                        "borderRightWidth": 1,
                        "borderStyle": "solid",
                        "borderTopColor": "#d1d5db",
                        "borderTopLeftRadius": 8,
                        "borderTopRightRadius": 8,
                        "borderTopWidth": 1,
                        "color": "#374151",
                        "flex": 1,
                        "fontFamily": "Inter_400Regular",
                        "fontSize": 14,
                        "height": 45,
                        "minWidth": 0,
                        "paddingBottom": 12,
                        "paddingLeft": 54,
                        "paddingRight": 16,
                        "paddingTop": 12,
                        "position": "relative",
                      }
                    }
                    testID="trigger-input"
                    textContentType="dateTime"
                  />
                </View>
              </View>
            </View>
            <View
              style={[]}
            >
              <View
                accessibilityHint="Select your gender for your profile"
                aria-autocomplete="none"
                aria-expanded={false}
                aria-label="Gender"
                focusVisibleStyle={
                  {
                    "outlineColor": "$outlineColor",
                    "outlineStyle": "solid",
                    "outlineWidth": 2,
                  }
                }
                focusable={true}
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                role="combobox"
                style={
                  {
                    "alignItems": "center",
                    "backgroundColor": "hsla(0, 0%, 100%, 1)",
                    "borderBottomColor": "hsla(0, 0%, 91%, 1)",
                    "borderBottomLeftRadius": 8,
                    "borderBottomRightRadius": 8,
                    "borderBottomWidth": 1,
                    "borderLeftColor": "hsla(0, 0%, 91%, 1)",
                    "borderLeftWidth": 1,
                    "borderRightColor": "hsla(0, 0%, 91%, 1)",
                    "borderRightWidth": 1,
                    "borderStyle": "solid",
                    "borderTopColor": "hsla(0, 0%, 91%, 1)",
                    "borderTopLeftRadius": 8,
                    "borderTopRightRadius": 8,
                    "borderTopWidth": 1,
                    "flexDirection": "row",
                    "flexWrap": "nowrap",
                    "height": 48,
                    "justifyContent": "space-between",
                    "maxWidth": "100%",
                    "minHeight": 16,
                    "overflow": "hidden",
                    "paddingBottom": 8,
                    "paddingLeft": 8,
                    "paddingRight": 8,
                    "paddingTop": 8,
                    "width": "100%",
                  }
                }
                testID="trigger"
                type="button"
              >
                <Text
                  lineBreakMode="clip"
                  numberOfLines={1}
                  pointerEvents="none"
                  style={
                    {
                      "color": "#374151",
                      "fontFamily": "Inter_400Regular",
                      "fontSize": 14,
                      "lineHeight": 24,
                      "opacity": 1,
                      "userSelect": "none",
                      "width": "100%",
                    }
                  }
                  suppressHighlighting={true}
                >
                  Female
                </Text>
                <View
                  pointerEvents="none"
                  style={
                    {
                      "height": 16,
                      "minHeight": 16,
                      "minWidth": 16,
                      "width": 16,
                    }
                  }
                />
                <RNSVGSvgView
                  align="xMidYMid"
                  bbHeight="24"
                  bbWidth="24"
                  fill="none"
                  focusable={false}
                  height="24"
                  meetOrSlice={0}
                  minX={0}
                  minY={0}
                  size={16}
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                  style={
                    [
                      {
                        "backgroundColor": "transparent",
                        "borderWidth": 0,
                      },
                      {
                        "flex": 0,
                        "height": 24,
                        "width": 24,
                      },
                    ]
                  }
                  vbHeight={24}
                  vbWidth={24}
                  width="24"
                >
                  <RNSVGGroup
                    fill={null}
                    propList={
                      [
                        "fill",
                        "stroke",
                      ]
                    }
                    stroke={
                      {
                        "type": 2,
                      }
                    }
                  >
                    <RNSVGPath
                      d="m6 9 6 6 6-6"
                      fill={
                        {
                          "payload": 4278190080,
                          "type": 0,
                        }
                      }
                    />
                  </RNSVGGroup>
                </RNSVGSvgView>
              </View>
            </View>
          </View>
          <View
            accessibilityHint="Submits your profile information"
            aria-label="Save profile"
            focusVisibleStyle={
              {
                "borderColor": "$background",
              }
            }
            pointerEvents="none"
            role="button"
            style={
              {
                "alignItems": "center",
                "backgroundColor": "#1c2a3a",
                "borderBottomColor": "transparent",
                "borderBottomLeftRadius": 60,
                "borderBottomRightRadius": 60,
                "borderBottomWidth": 1,
                "borderLeftColor": "transparent",
                "borderLeftWidth": 1,
                "borderRightColor": "transparent",
                "borderRightWidth": 1,
                "borderStyle": "solid",
                "borderTopColor": "transparent",
                "borderTopLeftRadius": 60,
                "borderTopRightRadius": 60,
                "borderTopWidth": 1,
                "color": "#ffffff",
                "flexDirection": "row",
                "flexWrap": "nowrap",
                "fontWeight": "500",
                "height": 48,
                "justifyContent": "center",
                "opacity": 0.75,
                "paddingBottom": 12,
                "paddingLeft": 20,
                "paddingRight": 20,
                "paddingTop": 12,
              }
            }
          >
            <Text
              lineBreakMode="clip"
              numberOfLines={1}
              style={
                {
                  "color": "#ffffff",
                  "flexGrow": 0,
                  "flexShrink": 1,
                  "fontFamily": "Inter_500Medium",
                  "fontSize": 14,
                  "lineHeight": 21,
                  "userSelect": "none",
                }
              }
              suppressHighlighting={true}
            >
              Save
            </Text>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </View>,
  <View
    pointerEvents="box-none"
    style={
      {
        "bottom": 0,
        "flexDirection": "column",
        "left": 0,
        "maxWidth": "100%",
        "position": "absolute",
        "right": 0,
        "top": 0,
        "zIndex": 110001,
      }
    }
  >
    <View
      onLayout={[Function]}
      style={
        {
          "bottom": 0,
          "left": 0,
          "opacity": 0,
          "pointerEvents": "none",
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
    <View
      animation={null}
      collapsable={false}
      disableClassName={true}
      onLayout={[Function]}
      style={
        [
          {
            "height": undefined,
            "minHeight": undefined,
            "opacity": 0,
            "pointerEvents": "none",
            "position": "absolute",
            "width": "100%",
            "zIndex": 100000,
          },
          {
            "transform": [
              {
                "translateY": 10000.1,
              },
            ],
          },
        ]
      }
    >
      <View
        accessibilityHint="Choose from a list of gender"
        aria-label="Select gender"
        pointerEvents="none"
        role="dialog"
        style={
          {
            "backgroundColor": "hsla(0, 0%, 100%, 1)",
            "borderTopLeftRadius": 34,
            "borderTopRightRadius": 34,
            "flex": 0,
            "flexDirection": "column",
            "gap": 24,
            "height": 200,
            "maxHeight": "100%",
            "overflow": "hidden",
            "paddingBottom": 24,
            "paddingLeft": 24,
            "paddingRight": 24,
            "paddingTop": 24,
            "width": "100%",
          }
        }
      >
        <RCTScrollView
          onMoveShouldSetResponder={[Function]}
          onResponderRelease={[Function]}
          onScroll={[Function]}
          onStartShouldSetResponder={[Function]}
          scrollEnabled={true}
          scrollEventThrottle={8}
          style={
            {
              "flex": 1,
            }
          }
        >
          <View>
            <View
              aria-labelledby=":r1o:"
              role="group"
              style={
                {
                  "flexDirection": "column",
                  "width": "100%",
                }
              }
            >
              <View
                id=":r1o:"
                style={
                  {
                    "alignItems": "center",
                    "backgroundColor": "hsla(0, 0%, 100%, 1)",
                    "borderBottomColor": "hsla(0, 0%, 91%, 1)",
                    "borderLeftColor": "hsla(0, 0%, 91%, 1)",
                    "borderRightColor": "hsla(0, 0%, 91%, 1)",
                    "borderTopColor": "hsla(0, 0%, 91%, 1)",
                    "flexDirection": "row",
                    "flexWrap": "nowrap",
                    "justifyContent": "space-between",
                    "maxWidth": "100%",
                    "minHeight": 16,
                    "overflow": "hidden",
                    "paddingBottom": 8,
                    "paddingLeft": 8,
                    "paddingRight": 8,
                    "paddingTop": 8,
                    "width": "100%",
                  }
                }
              >
                <Text
                  lineBreakMode="clip"
                  numberOfLines={1}
                  style={
                    {
                      "color": "#1c2a3a",
                      "flexGrow": 1,
                      "flexShrink": 1,
                      "fontFamily": "Inter_700Bold",
                      "fontSize": 16,
                      "lineHeight": 24,
                    }
                  }
                  suppressHighlighting={true}
                >
                  Gender
                </Text>
              </View>
              <View
                aria-labelledby=":r21:"
                aria-selected={false}
                focusVisibleStyle={
                  {
                    "outlineColor": "$outlineColor",
                    "outlineStyle": "solid",
                    "outlineWidth": 1,
                  }
                }
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  {
                    "alignItems": "center",
                    "backgroundColor": "hsla(0, 0%, 100%, 1)",
                    "borderBottomColor": "hsla(0, 0%, 91%, 1)",
                    "borderLeftColor": "hsla(0, 0%, 91%, 1)",
                    "borderRightColor": "hsla(0, 0%, 91%, 1)",
                    "borderTopColor": "hsla(0, 0%, 91%, 1)",
                    "flexDirection": "row",
                    "flexWrap": "nowrap",
                    "justifyContent": "space-between",
                    "maxWidth": "100%",
                    "minHeight": 16,
                    "overflow": "hidden",
                    "paddingBottom": 8,
                    "paddingLeft": 8,
                    "paddingRight": 8,
                    "paddingTop": 8,
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <Text
                  id=":r21:"
                  lineBreakMode="clip"
                  numberOfLines={1}
                  style={
                    {
                      "color": "#1c2a3a",
                      "fontFamily": "Inter_400Regular",
                      "fontSize": 16,
                      "lineHeight": 24,
                      "userSelect": "none",
                    }
                  }
                  suppressHighlighting={true}
                >
                  Male
                </Text>
              </View>
              <View
                aria-labelledby=":r27:"
                aria-selected={true}
                focusVisibleStyle={
                  {
                    "outlineColor": "$outlineColor",
                    "outlineStyle": "solid",
                    "outlineWidth": 1,
                  }
                }
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  {
                    "alignItems": "center",
                    "backgroundColor": "hsla(0, 0%, 100%, 1)",
                    "borderBottomColor": "hsla(0, 0%, 91%, 1)",
                    "borderLeftColor": "hsla(0, 0%, 91%, 1)",
                    "borderRightColor": "hsla(0, 0%, 91%, 1)",
                    "borderTopColor": "hsla(0, 0%, 91%, 1)",
                    "flexDirection": "row",
                    "flexWrap": "nowrap",
                    "justifyContent": "space-between",
                    "maxWidth": "100%",
                    "minHeight": 16,
                    "overflow": "hidden",
                    "paddingBottom": 8,
                    "paddingLeft": 8,
                    "paddingRight": 8,
                    "paddingTop": 8,
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <Text
                  id=":r27:"
                  lineBreakMode="clip"
                  numberOfLines={1}
                  style={
                    {
                      "color": "#1c2a3a",
                      "fontFamily": "Inter_400Regular",
                      "fontSize": 16,
                      "lineHeight": 24,
                      "userSelect": "none",
                    }
                  }
                  suppressHighlighting={true}
                >
                  Female
                </Text>
                <View
                  aria-hidden={true}
                  style={
                    {
                      "flexDirection": "row",
                      "marginLeft": "auto",
                    }
                  }
                >
                  <RNSVGSvgView
                    align="xMidYMid"
                    bbHeight="24"
                    bbWidth={16}
                    fill="none"
                    focusable={false}
                    height="24"
                    meetOrSlice={0}
                    minX={0}
                    minY={0}
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    style={
                      [
                        {
                          "backgroundColor": "transparent",
                          "borderWidth": 0,
                        },
                        {
                          "flex": 0,
                          "height": 24,
                          "width": 16,
                        },
                      ]
                    }
                    vbHeight={24}
                    vbWidth={24}
                    width={16}
                  >
                    <RNSVGGroup
                      fill={null}
                      propList={
                        [
                          "fill",
                          "stroke",
                        ]
                      }
                      stroke={
                        {
                          "type": 2,
                        }
                      }
                    >
                      <RNSVGPath
                        d="M20 6 9 17l-5-5"
                        fill={
                          {
                            "payload": 4278190080,
                            "type": 0,
                          }
                        }
                      />
                    </RNSVGGroup>
                  </RNSVGSvgView>
                </View>
              </View>
            </View>
          </View>
        </RCTScrollView>
      </View>
      <View
        accessibilityHint="Choose from a list of gender"
        aria-label="Select gender"
        role="dialog"
        style={
          {
            "backgroundColor": "hsla(0, 0%, 100%, 1)",
            "borderBottomLeftRadius": 0,
            "borderBottomRightRadius": 0,
            "borderBottomWidth": 0,
            "borderLeftWidth": 0,
            "borderRightWidth": 0,
            "borderStyle": "solid",
            "borderTopLeftRadius": 0,
            "borderTopRightRadius": 0,
            "borderTopWidth": 0,
            "bottom": "-100%",
            "flex": 1,
            "flexDirection": "column",
            "gap": 24,
            "height": 0,
            "left": 0,
            "maxHeight": "100%",
            "overflow": "hidden",
            "paddingBottom": 24,
            "paddingLeft": 24,
            "paddingRight": 24,
            "paddingTop": 24,
            "position": "absolute",
            "right": 0,
            "shadowColor": undefined,
            "shadowOffset": {
              "height": 0,
              "width": 0,
            },
            "shadowOpacity": 0,
            "shadowRadius": 0,
            "width": "100%",
            "zIndex": -1,
          }
        }
      />
    </View>
  </View>,
]
`;
